test/test_helpers/utilities.bash

```diff
@@ -57,3 +57,23 @@ assert_equal() {
 	fi
 }
 
+assert_not_empty() {
+	EMPTY=(\"\")
```

Comment:
I tried just doing a check like `if [[ -z "$1" ]]` or `if [[ -n "$1" ]]` but every variation I tried kept seeing this as a not-empty value, even if the outputted value in the echo showed `"value : """`